zingo-memo/src/lib.rs

```diff
@@ -11,159 +11,44 @@ use zcash_encoding::{CompactSize, Vector};
 #[non_exhaustive]
 #[derive(Debug)]
 pub enum ParsedMemo {
-    Version0 { uas: Vec<UnifiedAddress> },
```

Comment:
Isn't the above doc-comment out of date?   Don't we need a brief description of a `txindex-memo` (or whatever we want to call it)?

Reply:
Oh, yes, it is!